Tasks/NuGetCommandV2/task.json

```diff
@@ -9,7 +9,7 @@
     "version": {
         "Major": 2,
         "Minor": 147,
-        "Patch": 1
```

Comment:
Are there any other tasks that can call that packaging-common codepath?

Reply:
Does the .NET CLI task use it to configure NuGet.config (not sure if that is relevant in this context though).

Reply:
The NuGetCommandV2 is the only task that uses the cred provider and therefore the only task that will call isCredentialProviderEnabled. Dotnet tasks handle auth a bit differently - I believe the task adds the creds to a temp nuget.config so there's no need for a cred provider.